cli/azd/pkg/commands/pipeline/pipeline_manager.go

```diff
@@ -165,11 +166,16 @@ func (i *PipelineManager) pushGitRepo(ctx context.Context, currentBranch string)
 
 	console.Message(ctx, "Pushing changes")
 
-	if err := gitCli.PushUpstream(ctx, i.AzdCtx.ProjectDirectory(), i.PipelineRemoteName, currentBranch); err != nil {
-		return fmt.Errorf("pushing changes: %w", err)
-	}
-
-	return nil
+	// If user has a git credential manager with some cached credentials
+	// and the credentials are rotated, the push operation will fail and the credential manager would remote the cache
```

Comment:
Does this meant to say `remove`?
```suggestion
	// and the credentials are rotated, the push operation will fail and the credential manager would remove the cache
```